src/api/define.js

```diff
@@ -122,13 +133,16 @@ export default function (name, opts) {
   Ctor[$props] = createInitProps(Ctor);
   Ctor[$renderer] = createRenderer(Ctor);
 
-  if (customElementsV1) {
-    window.customElements.define(uniqueName, Ctor, { extends: Ctor.extends });
-  } else if (customElementsV0) {
-    return document.registerElement(uniqueName, Ctor);
-  } else {
-    throw new Error('Skate requires native custom element support or a polyfill.');
+  let res;
+  try {
```

Comment:
That would break those wanting to use standard HTML, which is kinda the foundation of web components.